src/main/asciidoc/override/eventbus.adoc

```diff
@@ -18,6 +18,14 @@ you don't have to specify the codec on each send in the delivery options:
 {@link docoverride.eventbus.Examples#example11}
 ----
 
+Default message codec performs a lookup based on the class of the body.
+
+This lookup happens after the system codecs are examined, i.e a default codec
+for `io.vertx.core.buffer.Buffer` will not be used.
+
+Since Vert.x 4.2, the lookup resolves on the super class of the body class when the body class does not have a codec, recursively yet
+excluding `java.lang.Object`.
+
```

Comment:
Should also mention that generics are not taken into consideration when looking for the codec.